### DISCUSSION TITLE
Error while View Metrics

### DISCUSSION THREAD

**Discussion by ImranDJCS** [DaxStudio/DaxStudio#623] [General]
Trying to connect with my shared dataset in Power BI Service using DAX Studio.
It connects successfully, but when I click on View Metrics, after a while it comes up with an error.
See attached.

Thank you

![View Metrics - error](https://user-images.githubusercontent.com/83199486/118575297-7d188300-b7c9-11eb-86ba-fe07b214e745.PNG)

**Comment by dgosbell** [comment]
What do you mean by "shared dataset"? If you are connecting to the XMLA endpoint of a premium capacity this should work. If you are trying to connect to a pro workspace I'm not sure how you are connecting since Microsoft does not support connections to pro workspaces by third party tools. Are you maybe using a local composite model with the new "DirectQuery over Power BI " feature? 

Can you collect more information by closing down DAX Studio, then start it up while holding down the SHIFT key to enable the [diagnostic logging](https://daxstudio.org/documentation/troubleshooting/logging-settings/)

Can you collect more information by closing down DAX Studio, then start it up while holding down the SHIFT key to enable the [diagnostic logging](https://daxstudio.org/documentation/troubleshooting/logging-settings/), then connect to your model and reproduce this issue, then close DAX Studio and upload the log file. You cannot upload files to a discussion, but I've created a new issue for this problem at #624 so you should be able to upload the log file there.

**Comment by ImranDJCS** [comment]
I created a composite data model in a Premium capacity, which is being used to create thin reports.
Log file loaded to specified space.
Thank you
